SU2_CFD/src/SU2_CFD.cpp

```diff
@@ -161,19 +152,7 @@ int main(int argc, char *argv[]) {
     /*--- Multi-zone problem: instantiate the multi-zone driver class by default
     or a specialized driver class for a particular multi-physics problem. ---*/
 
-    if (config->GetDiscrete_Adjoint()) {
-
-      if (turbo) {
-
-        driver = new CDiscAdjTurbomachineryDriver(config_file_name, nZone, nDim, periodic, MPICommunicator);
-
-      } else {
-
-        driver = new CDiscAdjFluidDriver(config_file_name, nZone, nDim, periodic, MPICommunicator);
-        
-      }
-
-    } else if (turbo) {
+    if (turbo) {
```

Comment:
Will the remaining capability from all other drivers (turbo, HB, etc) be absorbed into the primal and adjoint single/multizone drivers such that they can be removed eventually?